deploy/kessel-inventory-ephem-w-debezium.yaml

```diff
@@ -164,7 +186,13 @@ objects:
       generation: 1
       name: inventory-kafka-connect
     spec:
-      bootstrapServers: inventory-kafka-kafka-bootstrap:9092
+      bootstrapServers: inventory-kafka-kafka-bootstrap:9094
+      authentication:
+        type: scram-sha-512
+        username: inventory-consumer
+        passwordSecret:
+          secretName: inventory-consumer
+          password: password
```

Comment:
can't wait for security scanning to flag this false positive 😅

Reply:
it already has with gitguardian hahaha